src/Views/AlertList/Types.elm

```diff
@@ -0,0 +1,14 @@
+module Views.AlertList.Types exposing (AlertListMsg(..), Route(..))
+
+import Utils.Types exposing (ApiData, Filter)
+import Alerts.Types exposing (Alert, AlertGroup)
+
+
+type Route
+    = Receiver (Maybe String) (Maybe Bool) (Maybe String)
```

Comment:
Maybe something for later, but this is a nicer type for `Route` because it names the attributes:

```
type alias Route = {
  receiver: Maybe String,
  silenced: Maybe Bool,
  filter: Maybe String
}
```